tensorboard/webapp/widgets/card_fob/card_fob_controller_test.ts

```diff
@@ -668,6 +757,52 @@ describe('card_fob_controller', () => {
       ).toEqual(3);
       expect(onTimeSelectionChanged).toHaveBeenCalledTimes(0);
     });
+
+    it('adds and removes event listener', () => {
```

Comment:
I feel like your first test was sufficient to cover everything. The relevant code isn't different for horizontal vs vertical. If you agree with this feeling, maybe just add a `describe('dragging')` as a sibling to `describe('vertical dragging')` and `describe('horizontal dragging')` and put the one and only copy of this test there.

Reply:
Done.